### PR TITLE
feat: adds rsktestnet config

### PR DESCRIPTION
**Pull request by jurajpiar**

### CODE REVIEW

File: src/ui-config.json [hunk 1]
```diff
@@ -4,5 +4,11 @@
         "rnsDotRskOwner": "0x4bf749ec68270027C5910220CEAB30Cc284c7BA2",
         "rnsReverseRegistrar": "0xe982E462b094850F12AF94d21D470e21bE9D0E9C",
```
Comment (julianmrodri) [inline src/ui-config.json:5]: Shouldnt we fix this other configuration? To avoid confusion?

Comment (julianmrodri) [reply src/ui-config.json:5]: They should both have the same varilables (I guess the one for Testnet are the correct)

File: src/ui-config.json [hunk 2]
```diff
@@ -4,5 +4,11 @@
...
         "marketplace": "0x22d5C8BdD4346b390014a07109a8F830094d4abf"
-    }
-}
\ No newline at end of file
+    },
+    "rsktestnet": {
+        "rif": "0x19f64674D8a5b4e652319F5e239EFd3bc969a1FE",
+        "rnsDotRskOwner": "0xca0a477e19bac7e0e172ccfd2e3c28a7200bdb71",
+        "rnsNameResolver": "0x8587385ad60038bB181aFfDF687c4D1B80C4787e",
```
Comment (vojtechsimetka) [inline src/ui-config.json:11]:
```suggestion
        "rnsReverseRegistrar": "0x8587385ad60038bB181aFfDF687c4D1B80C4787e",
```

File: src/ui-config.json (hunk 1 above)
Comment (vojtechsimetka) [reply src/ui-config.json:5]: Good eye

Comment (vojtechsimetka) [reply src/ui-config.json:5]: Even though the correct name would actually be `rnsNameResolver`

Comment (julianmrodri) [reply src/ui-config.json:5]: I thought rnsNameResolver was the one being used

**Pull request merged**